service/src/java/org/apache/hive/service/cli/HiveSQLException.java

```diff
@@ -150,4 +185,11 @@ public static TStatus toTStatus(Exception e) {
     return tStatus;
   }
 
+  private static String getErrorMsg(String message, String queryId) {
```

Comment:
i think it could be done simpler, just override the Throwable#getMessage
````
@Override
String getMessage(){
String errorMsg = super.getMessage();
 if (!errorMsg.contains(ServiceContext.QUERY_ID)) {
      return String.format("%1$s; %2$s: %3$s", errorMsg, ServiceContext.QUERY_ID, queryId);
    } else {
      return errorMsg;
    }
}
````